2996_reflection/reflection.md

```diff
@@ -1205,17 +1207,17 @@ using fn_t = metatype<^std::meta::is_function>;
 
 // Example of a function overloaded for different "types" of reflections.
 void PrintKind(type_t) { std::println("type"); }
-void PrintKind(fn_t) { std::println("function"); }
```

Comment:
`template_t` needs to be introduced in this example somewhere.

Reply:
Whoops, thanks. Done.

I changed this after a helpful person pointed out that it's illegal to name `main` :-P

Reply:
Still this one. Guessing just a `using template_t = metatype<^std::meta::is_template>;`?

Reply:
Ugh, thought I fixed this! Thanks, done.